resources/install.conf

```diff
@@ -10,5 +10,4 @@ Which disk is the root disk = sd0
 Use (W)hole disk MBR, whole disk (G)PT or (E)dit = g
 URL to autopartitioning template for disklabel = {{ DISKLABEL_TEMPLATE }}
 Http Server = cdn.openbsd.org
-Set name(s) = -game* -x* -man*
```

Comment:
Are games and man pages really required? I'm trying to make the VM image as small as possible.

Reply:
It should be fine without them, if the image is only for CI runners, updated.

Reply:
Yes, that's the intention.